lua/lush_theme/lush_template.lua

```diff
@@ -50,138 +50,137 @@ local hsl = lush.hsl
 ---@diagnostic disable: undefined-global
 local theme = lush(function()
   return {
-    -- The following are all the Neovim default highlight groups from the docs
-    -- as of 0.5.0-nightly-446, to aid your theme creation. Your themes should
-    -- probably style all of these at a bare minimum.
+    -- The following are the Neovim default highlight groups from the docs.
+    -- Your themes should probably style all of these at a bare minimum.
```

Comment:
How about something like:

```
-- The following are the Neovim (as of 0.7) highlight groups, mostly used
-- for styling UI elements.
-- Comment them out and add your own properties to override the defaults.
-- An empty definition `{}` will clear all styling, leaving elements looking
-- like the 'Normal' group.
-- To be able to link to a group, it must already be defined, so you may have
-- to reorder this list as you go.
--
-- See :h highlight-groups
--
-- ColorColumn  { }, -- Columns set with 'colorcolumn'
-- Conceal      { }, -- Placeholder characters substituted for concealed text (see 'conceallevel')
-- Cursor       { }, -- Character under the cursor
…
```
I don't think we even need to suggest to colorscheme creators that they better had set all these.

Reply:
Just tweak `0.7` to the full `:version` line, as many people run nightly, I'd just prefer to have no obscurity around which version was dumped.

Other than that LGreatTM :+1:

Reply:
I think I totally misunderstood the purpose of having the version listed. It's not about finding the earliest version where these highlights exist, but about giving the maintainer an idea how recent the dump is – so having `0.8-nightly-456` is "better" than `0.7-stable`, right?

Reply:
Yep just stick in whatever your version is.

> the newer the better

Within reason. I think it's best to keep bleeding edge groups out as they may change (so no "new-to-nightly" groups), I believe the new ones you're adding are all new-to-0.7 so we're good.

Reply:
OK, I'll then list my `0.8` nightly build there, as these groups haven't changed so far since `0.7` stable. Only the Tree-Sitter groups are newer, but that's still the external plugin, not part of the nvim source.